spec/controllers/api/v1/accounts/statuses_controller_spec.rb

```diff
@@ -48,6 +47,25 @@
       end
     end
 
+    context 'with exclude sensitive' do
+      let!(:sensitive_status) { Fabricate(:status, sensitive: true, account: user.account) }
+      let!(:insensitive_status) { Fabricate(:status, account: user.account) }
+
+      before do
+        get :index, params: { account_id: user.account.id, exclude_sensitive: true }
+      end
+
+      it 'returns http success' do
+        expect(response).to have_http_status(200)
+      end
+
+      it 'does not return posts marked sensitive' do
+        json = body_as_json
+        post_ids = json.map { |item| item[:id].to_i }.sort
+        expect(post_ids).to eq [insensitive_status.id]
```

Comment:
Minor style thing - I'd avoid the local var here and just do `expect(body_as_json).to include(...)` sort of thing as one-liner.